File: app/src/main/java/catchla/yep/view/iface/TintedStatusLayout.java
/*
 * Copyright (c) 2015. Catch Inc,
 */

package catchla.yep.view.iface;

import android.graphics.Rect;

/**
 * Created by mariotaku on 15/4/27.
 */
public interface TintedStatusLayout extends IExtendedView {
    void setColor(int color);

    void setColor(int color, int alpha);

    void setDrawColor(boolean color);

    void setDrawShadow(boolean shadow);

    void setFactor(float f);

    void setShadowColor(int color);

    void setSetPaddingEnabled(boolean enabled);

    void getSystemWindowsInsets(Rect insets);
}

File: src/main/java/catchla/yep/view/iface/TintedStatusLayout.kt
/*
 * Copyright (c) 2015. Catch Inc,
 */

package catchla.yep.view.iface

/**
 * Created by mariotaku on 15/4/27.
 */
interface TintedStatusLayout : IExtendedView {
    fun setStatusBarColorDarken(color: Int)

    var setPaddingEnabled: Boolean
}
